### ISSUE TITLE
Opera Support

### ISSUE THREAD

**Issue by kmausolf** [lrkissling/giant-bomb-qol#47]
Opera's new Chromium-based browser is pretty hot, and I'd like to port the extension to it.  Should be pretty straightforward, as it's based on Chromium.

**Comment by kmausolf** [comment]
Can you assign me, @lrkissling ?

**Comment by kmausolf** [comment]
https://dev.opera.com/extensions/publishing-guidelines/
https://dev.opera.com/extensions/basics/

Saving these here for later.

**Comment by lrkissling** [comment]
https://dev.opera.com/extensions/architecture-overview/

This page is useful for listing things they discourage or disallow. Of concern is that they seriously recommend against using setInterval(), which QB QoL uses for the live stream indicator.

**Comment by lrkissling** [comment]
If we add Opera support, I will finally want to create a build script so I don't have to manually create the build for each browser. I will create a separate issue for that.

**Comment by kmausolf** [comment]
That's a great idea.  Also, do we have any kind of semantic versioning?

**Comment by lrkissling** [comment]
No, I've never given versioning that much thought. Going forward maybe we could do whole number versions for new features. Use x.1, x.2, etc for bug fixes, new emotes, and super small tweaks.

**Comment by kmausolf** [comment]
https://semver.org/

This is an excellent primer for semver.  It can be useful for declaring when a change is incompatible with previous versions, and so on.
